src/App.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 import { drawContributions } from "github-contributions-canvas";
-import { download } from "./utils/export";
+import { download, onShareTwitter } from "./utils/export";
```

Comment:
I'm not sure if you should change the name of this method :) only callbacks inside the component should begin with `on` prefix. From my point of view, `postMediaTweet` or `postTweet` should be a better name.

Naming is really hard :)

Reply:
I agree. @konojunya let's change the name to "`uploadToTwitter`"

Reply:
ok!
Thank you for careful code review :)